File: gmall0715-canal/src/main/java/com/cy/gmall0715/canal/util/PropertiesUtil.java
package com.cy.gmall0715.canal.util;

import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Properties;

/**
 * @author cy
 * @create 2019-12-30 18:45
 */
public class PropertiesUtil {
    public static Properties load(String propertieName) throws IOException {
        Properties properties = new Properties();
        properties.load(new InputStreamReader(Thread.currentThread().getContextClassLoader().getResourceAsStream(propertieName),"UTF-8"));
        return properties;
    }
}

File: gmall0715-realtime/src/main/scala/util/PropertiesUtil.scala
package util

import java.io.InputStreamReader
import java.util.Properties

/**
  * @author cy
  * @create 2019-12-15 17:12
  */
object PropertiesUtil {
  def main(args: Array[String]): Unit = {
    val properties: Properties = PropertiesUtil.load("config.properties")

    println(properties.getProperty("kafka.broker.list"))
  }

  def load(propertieName:String): Properties ={
    val prop=new Properties();
    prop.load(new InputStreamReader(Thread.currentThread().getContextClassLoader.getResourceAsStream(propertieName) , "UTF-8"))
    prop
  }

}
